test/suites-ostree/failed-upgrade.robot

```diff
@@ -31,7 +31,8 @@ New Deployment Is Consistently Unhealthy And Rolls Back
     TestAgent.Add Action For Next Deployment    every    ${REASON}
     Deploy Commit Expecting A Rollback    ${FAILING_REF}
     Wait For MicroShift Service
-    Backup Should Exist    ${backup}
+    # backup logic is only present in 4.14+
+    IF    ${backup} != "4.13"    Backup Should Exist    ${backup}
```

Comment:
Looks like CI fails because:
```
Evaluating IF condition failed: Evaluating expression 'rhel-16a972f4e385541d7cdc2c4e9071ce6032f1042753eaab53e2a1eca084aa996c.0_fc34f14811a846b7a91a111f9dfab4ad != "4.13"' failed: SyntaxError: invalid syntax (<string>, line 1)
```
Maybe we need to wrap the var into quotes.

However, I think we can drop "Backup Should Exist" only in case of 4.13 & "prevent backup".
For `4.14`, even if the 2nd deployment fails to create a backup, it should be created after rolling back.


```suggestion
    # backup logic is only present in 4.14+
    IF    "${backup}" != "4.13" and "${REASON}" != "prevent_backup"    Backup Should Exist    ${backup}
```

BTW. I see I missed the filename of https://github.com/openshift/microshift/blob/main/test/scenarios/el92-src%40upgrade-fails-cannot-backup.sh, it should be "upgrade-fails.sh" without cannot-backup...